src/views/Generator/Thresholds/ThresholdsDialog.tsx

```diff
@@ -0,0 +1,139 @@
+import { ThresholdDataSchema } from '@/schemas/generator'
+import { useGeneratorStore } from '@/store/generator'
+import { Threshold, ThresholdData } from '@/types/thresholds'
+import { css } from '@emotion/react'
+import { zodResolver } from '@hookform/resolvers/zod'
+import {
+  Dialog,
+  Table,
+  Button,
+  Text,
+  Link as RadixLink,
+} from '@radix-ui/themes'
+import { useCallback, useEffect } from 'react'
+import { useForm, useFieldArray, FormProvider } from 'react-hook-form'
+import { ThresholdRow } from './ThresholdRow'
+import { uniqueId } from 'lodash-es'
```

Comment:
Nit(because there's one more place that uses this import), but have you considered `crypto.randomUUID()`?